src/library/dao/trackdao.cpp

```diff
@@ -353,26 +353,31 @@ void TrackDAO::addTracksPrepare() {
 
     m_pQueryTrackLocationSelect->prepare("SELECT id FROM track_locations WHERE location=:location");
 
-    m_pQueryLibraryInsert->prepare("INSERT INTO library "
+    m_pQueryLibraryInsert->prepare(
+            "INSERT INTO library "
             "("
-            "artist,title,album,album_artist,year,genre,tracknumber,tracktotal,composer,"
-            "grouping,filetype,location,color,comment,url,rating,key,key_id,"
+            "artist,title,album,album_artist,year,genre,"
+            "tracknumber,tracktotal,composer,grouping,"
+            "filetype,location,color,comment,url,rating,key,key_id,"
             "cuepoint,bpm,replaygain,replaygain_peak,wavesummaryhex,"
             "timesplayed,played,mixxx_deleted,header_parsed,"
             "channels,samplerate,bitrate,duration,"
             "beats_version,beats_sub_version,beats,bpm_lock,"
             "keys_version,keys_sub_version,keys,"
-            "coverart_source,coverart_type,coverart_location,coverart_hash,"
+            "coverart_source,coverart_type,coverart_location,"
+            "coverart_color,coverart_digest,coverart_hash,"
             "datetime_added"
             ") VALUES ("
-            ":artist,:title,:album,:album_artist,:year,:genre,:tracknumber,:tracktotal,:composer,"
-            ":grouping,:filetype,:location,:color,:comment,:url,:rating,:key,:key_id,"
+            ":artist,:title,:album,:album_artist,:year,:genre,"
+            ":tracknumber,:tracktotal,:composer,:grouping,"
+            ":filetype,:location,:color,:comment,:url,:rating,:key,:key_id,"
             ":cuepoint,:bpm,:replaygain,:replaygain_peak,:wavesummaryhex,"
             ":timesplayed,:played,:mixxx_deleted,:header_parsed,"
             ":channels,:samplerate,:bitrate,:duration,"
             ":beats_version,:beats_sub_version,:beats,:bpm_lock,"
             ":keys_version,:keys_sub_version,:keys,"
-            ":coverart_source,:coverart_type,:coverart_location,:coverart_hash,"
+            ":coverart_source,:coverart_type,:coverart_location,"
+            ":coverart_color,:coverart_digest,:coverart_hash,"
             ":datetime_added"
```

Comment:
Should we put every item on a separate line for smaller diffs and less merge conflicts in the future?